desafio-02/maurilioSo/java/PrimeNumbers.java

```diff
@@ -0,0 +1,28 @@
+
+public class PrimeNumbers {
+
+	public static void main(String[] args) {
+		//O Programa gera uma lista com os numeros primos entre 1 e 10000.
+
+		int num = 2, max = 10000;
+
+		while(num < max) {
+			int test = 0;
+			for(int i = 2; i <= num / 2; i++) {
+				if(num % i == 0) {
```

Comment:
Sugestão: crie um método/função conforme os pontos a seguir:

- recebe o número atual como parâmetro
- verifica se o número recebido como parâmetro é primo
- retorna verdadeiro se o número for primo
- retorna falso se o número não for primo

Aí vc usa esse método para verificar se o teu número vai ou não ser exibido.